cmd/antrea-agent/agent.go

```diff
@@ -97,14 +97,19 @@ func run(o *Options) error {
 	}
 	defer ovsdbConnection.Close()
 
+	// Enable AntreaIPAM will set enableBridge to True. Currently only Linux+IPv4 is supported.
+	// AntreaIPAM works with system OVSDatapathType, noEncap and no Egress SNAT.
```

Comment:
what does it mean "works with no Egress SNAT"?

Reply:
Probably "and noEncap mode. Egress feature is not supported."

Reply:
Egress and SNAT is not supported currently.